src/17-infinite-discovery/01-user.ts

```diff
@@ -49,10 +49,10 @@ async function prepareCollection(className: UsersClassName) {
           "Artworks liked by this user. Used to calculate the user's vector.",
       },
       {
-        name: "dislikedArtworks",
+        name: "seenArtworks",
```

Comment:
There is an argument to be made that this isn't the best naming. Perhaps dismissedArtworks would be better as technically likedArtworks have also been seen.